app-ios/tutanota/Sources/Remote/RemoteBridge.swift

```diff
@@ -170,6 +172,18 @@ class RemoteBridge : NSObject, NativeInterface {
       }
     }
   }
+
+  func vacuumOfflineDatabase() {
```

Comment:
I would rather create it in `ViewController` and pass it here just for the bridge itself. Bridge shouldn't do anything besides bridging and `ViewController` has lifecycle methods and is generally a kitchen sink anyway.

Reply:
Ok, but than I need to expose the  IosSqlCipherFacade? Wanted to keep it private within the remoteBridge.

Reply:
The bridge shouldn't abstract anything inside of it besides the communication channel imo, viewController already holds all dependencies.

Reply:
Ok, done :+1: (Need to test on iOS again though)